.github/workflows/release.yml

```
@@ -0,0 +1,61 @@
+name: Weekly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Monday
+    - cron: "0 0 * * 1"
+
+jobs:
+  bump-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Create Tag
+        id: tag_version
+        run: |
+          # Extracts the latest version tag, bumps the minor version
+          LATEST_VERSION=$(git describe --tags `git rev-list --tags --max-count=1`)
```

We could also use the following to only consider tags that have been merged to `main`:
```suggestion
          LATEST_VERSION=$(git tag --sort=-authordate --merged=main | head --lines 1)
```

Looking at the output of `git rev-list --tags`,  I see a lot of weird tags like `v2.239.0-5-g85284e74`. (Can be checked with `for t in `git rev-list --tags --max-count=100`; do git describe --tags "$t"; done`.)
But I'm not sure this is actually a problem.